src/python/pants/backend/python/target_types.py

```diff
@@ -220,8 +220,8 @@ class PythonBinary(Target):
     """A Python target that can be converted into an executable PEX file.
 
     PEX files are self-contained executable files that contain a complete Python environment capable
-    of running the target. For more information about PEX files, see
-    https://www.pantsbuild.org/docs/pex-files.
+    of running the target. For more information about PEX files, see https://www.pantsbuild.org/docs
+    /pex-files.
```

Comment:
Won't this edit insert a space in the url?

Reply:
Yeah, but I have no idea how to get docformatter to not automatically fix it :/ And I don't know why docformatter is suddenly making the change.

I can play with it more.